### ISSUE TITLE
Sails Waterline “Or” clause not working

### ISSUE THREAD

**Issue by vijrej** [balderdashy/sails-mysql#242]
I tried a lot of time to figure out getting an OR clause working in sails without success.

I am using Sails-MySql adapter.

Have anyone of you done anything like this already? I would appreciate some help.

Basically this is what I want to do:

Do an OR clause on a set of fields along with an AND on another set of fields.

Something like this:
FdbDevice.find().where( or : [ { deviceCategory: “cardiology valve bioprosthesis” }, { deviceCategory: “nephrology haemodialysis catheter” } ] } )
                              .where (  { catalogNumber : “Z286004” },  { modelNumber: “Z286004” })
                              .exec

**Comment by sailsbot** [comment]
Thanks for posting, @vijrej.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
